ofl/braahone/METADATA.pb

```diff
@@ -0,0 +1,23 @@
+name: "Braah One"
+designer: "Ashish Kumar"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2023-03-24"
+fonts {
+  name: "Braah One"
+  style: "normal"
+  weight: 400
+  filename: "BraahOne-Regular.ttf"
+  post_script_name: "BraahOne-Regular"
+  full_name: "Braah One Regular"
+  copyright: "Copyright 2022 The Braah Project Authors (https://github.com/artandtype/Braah )"
+}
+subsets: "gurmukhi"
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "vietnamese"
+source {
+  repository_url: "https://github.com/artandtype/Braah"
+  commit: "d10ec8d1efc1d3ed756348ea4b94f71849f6ec26"
+}
```

Comment:
primary script to add here (https://googlefonts.github.io/gf-guide/metadata.html#primary-script)